boards/arm/rcar_h3ulcb/doc/rcar_h3ulcb.rst

```diff
@@ -122,60 +130,111 @@ Here is CN04 UART interface pinout (depending on your Kingfisher board version)
 | GND    | 9        | 6        |
 +--------+----------+----------+
 
+CAN :
+-----
```

Comment:
```suggestion
CAN
---
```

Reply:
In this case, should I also remove colons from other sections such as GPIO and UART ?

Reply:
Yes, : should be removed